docs/src/examples/gol_lazar_belta.jl

```diff
@@ -0,0 +1,164 @@
+using Test     #src
+# # Example: Gol, Lazar and Belta (2013)
+#
+#md # [![Binder](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/literate/literate_gol_lazar_belta.ipynb)
+#md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/literate/literate_gol_lazar_belta.ipynb)
+#
+# This example was borrowed from [[1, Example VIII.A]](#1) and tackles 
+# an optimal control for the hybrid system with state evolution governed by 
+#nb # ```math
+#nb # x(k+1) = \begin{bmatrix} 1 & 1 \\ 0 & 1 \end{bmatrix}x(k) + \begin{bmatrix} 0.5 \\ 1.0 \end{bmatrix} u(k) 
+#nb # ```
+#md # 
+#md # <img src="https://latex.codecogs.com/svg.latex?x(k&plus;1)&space;=&space;\begin{bmatrix}&space;1&space;&&space;1&space;\\&space;0&space;&&space;1&space;\end{bmatrix}x(k)&space;&plus;&space;\begin{bmatrix}&space;0.5&space;\\&space;1.0&space;\end{bmatrix}&space;u(k)" title="x(k+1) = \begin{bmatrix} 1 & 1 \\ 0 & 1 \end{bmatrix}x(k) + \begin{bmatrix} 0.5 \\ 1.0 \end{bmatrix} u(k)" />
+#md #
+# The goal is to take the state vector toward a target set **XT** by visiting one of the squares
+# **A** or **B** and avoiding the obstacles **O1** and **O2**
+
+
+# First, let us import [CDDLib](https://github.com/JuliaPolyhedra/CDDLib.jl/tree/v0.6.3), 
+# [GLPK](https://github.com/jump-dev/GLPK.jl), [OSQP](https://github.com/oxfordcontrol/OSQP.jl),
+# [JuMP](https://github.com/jump-dev/JuMP.jl), [Pavito](https://github.com/jump-dev/Pavito.jl)
+# and [Ipopt](https://github.com/jump-dev/Ipopt.jl)
+import CDDLib
+import GLPK
+import OSQP
+using JuMP
+import Pavito
+import Cbc
+import Ipopt
+
+# At this point we import Dionysos
+using Dionysos
+
+# And the file defining the hybrid system for this problem
+include(joinpath(dirname(dirname(pathof(Dionysos))), "examples", "gol_lazar_belta.jl"))
+
+# Now we instantiate our system using the function provided by [gol_lazar_belta.jl](@__REPO_ROOT_URL__/examples/gol_lazar_belta.jl)
+system = gol_lazar_belta(CDDLib.Library());
+
+
+# Then, we define initial conditions (continuous and discrete states) to this system
+# and set **N** as the search depth, i.e., the number of allowed time steps.
+
+x0 = [1.0, -6.0];
+q0 = 3;
+
+N = 11;
+
+# We instantiate our Optimal Control Problem by defining the state and transition costs
```

Comment:
Comment that `state_cost` is defined to be zero for each mode/discrete state of the system and `transition` cost is defined to be `u_1^2` which use `u' * Q * u` for `Q = ones(1, 1)`.

Reply:
Good! Just done it